xml/deployment_hwrecommend.xml

```diff
@@ -173,9 +173,13 @@
     <para>
      If the monitor nodes are on multiple subnets, for example they are located
      in different rooms and served by different switches, you need to specify
-     their public network address:
+     their public network address in CIDR notation:
     </para>
-<screen>&prompt.cephuser;ceph config set mon public_network <replaceable>MON_NETWORK</replaceable></screen>
+<screen>&prompt.cephuser;ceph config set mon public_network "<replaceable>MON_NETWORK_1</replaceable>, <replaceable>MON_NETWORK_2</replaceable>, <replaceable>MON_NETWORK_N</replaceable></screen>
+    <para>
+      For example:
+    </para>
+<screen>&prompt.cephuser;ceph config set mon public_network "1.3.4.0/24, 1.3.43.0/24, 1.3.42.0/24"</screen>
```

Comment:
Please change `"1.3.4.0/24, 1.3.43.0/24, 1.3.42.0/24"` to:

```
"192.168.1.0/24, 10.10.0.0/16"
```

Reply:
I considered that, but we've had this discussion before.

There's also the 172.16.0.0–172.31.255.255 range, of course.

Reply:
@lproven We cannot use `1.3.4.0/24, 1.3.43.0/24, 1.3.42.0/24` because it would look like we are plagiarizing https://access.redhat.com/solutions/1463363

;-)

Reply:
@lproven Of course, if you'd like to add `172.16.0.0–172.31.255.255` as well, please provide the CIDR equivalent.